script.js

```diff
@@ -1,4 +1,132 @@
+//2 players take turns. when player_1 click submit, roll 2 dice
+//Output shows player_1's two roll number, prompt player 1 to choose 1st or 2nd dice
+//Output shows player_1's final number, prompt player_2 to click submit to roll 2 dice
+//Output shows player_2's two roll numbers, prompt player 2 to choose 1st or 2nd dice
+//Output shows player_2's final number, shows who win, prompt to click submit to reset the game
+
+//CODE1: this is the basic codes
+
+var gameMode = "p1_rolldice"; //Five Modes: p1_rolldice, p1_sequence, p2_rolldice, p2_sequence, compare
+var p1_dice_array = [];
+var p2_dice_array = [];
+var myOutputValue = "";
+
+var rollDice = function () {
+  var dice_num = Math.floor(Math.random() * 5 + 1);
+  return dice_num;
+};
+
 var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+  if (gameMode == "p1_rolldice") {
+    var p1_d1 = rollDice();
+    var p1_d2 = rollDice();
+    p1_dice_array.push(p1_d1);
+    p1_dice_array.push(p1_d2);
+    myOutputValue =
+      "Welcome, Player 1 <br> You rolled " +
+      p1_dice_array[0] +
+      " for DICE ONE and " +
+      p1_dice_array[1] +
+      " for DICE TWO. <br> Choose the order of the dice by entering 1 or 2.";
+    console.log(
+      "p1 numbers:",
+      p1_dice_array[0],
+      p1_dice_array[1],
+      p1_d1,
+      p1_d2
+    );
+    gameMode = "p1_sequence";
```

Comment:
i've already spoke on my concerns about doing modes and states like this before, just thought to highlight that again 😄